### PR TITLE
ele VID fixes for DisappTrk track methods

### PR DESCRIPTION
**Pull request by bpf6qc**
Remove MVA and HEEP ID VIDs for electrons
Fix specific case for DisappTrks with electrons + tracks
Require vertexing for DisappTrks track::deltaRToClosest*Electron methods

### CODE REVIEW

File: Collections/src/Track.cc [hunk 1]
```diff
@@ -321,11 +322,81 @@ osu::Track::set_minDeltaRToElectrons (const edm::Handle<edm::View<TYPE(electrons
...
-    if((*vidTightMap) [(*electrons).refAt(iEle)] && (dR < deltaRToClosestTightElectron_  || deltaRToClosestTightElectron_  < 0.0)) deltaRToClosestTightElectron_  = dR;
-  }
+
+    bool passesVeto_dxy   = false, passesVeto_dz   = false;
+    bool passesLoose_dxy  = false, passesLoose_dz  = false;
+    bool passesMedium_dxy = false, passesMedium_dz = false;
+    bool passesTight_dxy  = false, passesTight_dz  = false;
+
+    // Note in below, these remain false if |eta| >= 2.5; thus an eta cut is also being applied here as intended
+#if CMSSW_VERSION_CODE < CMSSW_VERSION(9,4,0)
```
Comment (jalimena) [inline Collections/src/Track.cc]: L332-370: Why are all these dxy cut values hardcoded?

Comment (bpf6qc) [reply Collections/src/Track.cc]: The EGM POG doesn't include vertex requirements in the VIDs for various reasons, so these cuts need to be made explicitly (they were forgotten before this PR by mistake). Plus this isn't the normal framework "electrons" collection, so they can't be provided in the typical cut PSet fashion.

Are you suggesting however that these be external PSet parameters?

Comment (jalimena) [reply Collections/src/Track.cc]: Ah ok, thanks for the information. But yeah, wouldn't external PSet parameters be a good idea?

Comment (bpf6qc) [reply Collections/src/Track.cc]: I can do that; it'll look cleaner too.

**Comment by jalimena** [comment]
hi @bpf6qc , I'm going to run a test with this PR now to check that it doesn't break anything for displaced leptons. But also, what's the status of doing the modification we talked about above?

**Comment by bpf6qc** [comment]
@jalimena I haven't had the time to do that yet, but I intend to within the week.

**Comment by bpf6qc** [comment]
Testing recent commit...

**Comment by bpf6qc** [comment]
@jalimena I'm satisfied with my changes now. These methods are all behind
```#ifdef DISAPP_TRKS```
so the potential to affect other analyses should be zero by design.

**Comment by jalimena** [comment]
cool, looks good

**Pull request merged**